src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -121,12 +121,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 			// Show Disabled UI
 			setupUpdateOSUI()
 			didSetupUI = true
-			return true
+
+			// Return false, because if the app is disabled, we cannot handle URL ressources or user activity.
+			// More information: https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application
+			return false
 		}
 
-		// Check for any URLs passed into the app – most likely via scanning a QR code from event or antigen rapid test
-		// Route will be executed in 'applicationDidBecomeActive'
-		route = routeFromLaunchOptions(launchOptions)
+		// 'appLaunchedFromUserActivityURL' inidcates, if the app was launched through a QR-Code scan.
```

Comment:
```suggestion
		// 'appLaunchedFromUserActivityURL' inidcates, if the app was launched through a QR-Code scan, from the System Camera.
```
Thats correct right?